f5lbaasdriver/test/tempest/tests/api/test_l7policy_rules.py

```diff
@@ -265,67 +307,45 @@ def test_policy_reject_multi_policy_multi_rules(self):
 
         self._delete_l7rule(l7policy1.get('id'), self.rule1.get('id'))
         for bigip_client in self.bigip_clients:
-            assert bigip_client.policy_exists("wrapper_policy",
-                                              "Project_" +
-                                              self.project_tenant_id)
-            assert bigip_client.rule_exists("wrapper_policy",
-                                            "reject_1",
-                                            "Project_" +
-                                            self.project_tenant_id)
-            assert bigip_client.rule_exists("wrapper_policy",
-                                            "reject_2",
-                                            "Project_" +
-                                            self.project_tenant_id)
-            assert not bigip_client.rule_has_condition("wrapper_policy",
```

Comment:
ln 319 in the modified, I guess that I'm confused why this is a negative test case.  At lines 258 and 263 in the original code, we're introducing this rule.  Thus, it __should__ be there.  Yet, we're testing that it is __not__ there.

Reply:
If I change these to be all positive assertions, then the test passes:
```python
test_l7policy_rules.py::L7PolicyJSONReject::test_create_l7_reject_policy PASSED
test_l7policy_rules.py::L7PolicyJSONReject::test_policy_reject_header_contains PASSED
test_l7policy_rules.py::L7PolicyJSONReject::test_policy_reject_header_ends_with PASSED
test_l7policy_rules.py::L7PolicyJSONReject::test_policy_reject_many_rules PASSED
test_l7policy_rules.py::L7PolicyJSONReject::test_policy_reject_multi_policy_multi_rules PASSED
test_l7policy_rules.py::L7PolicyJSONReject::test_policy_reject_three_rules FAILED
test_l7policy_rules.py::TestL7PolicyTestJSONRedirectToUrl::test_create_l7_redirect_to_url_policy PASSED
test_l7policy_rules.py::TestL7PolicyTestJSONRedirectToUrl::test_policy_redirect_url_contains PASSED
test_l7policy_rules.py::L7PolicyJSONRedirectToPool::test_create_l7_redirect_to_pool_policy_file_type_contains PASSED
test_l7policy_rules.py::L7PolicyJSONRedirectToPool::test_create_l7_redirect_to_pool_policy_not_file_type PASSED
```